### DISCUSSION TITLE
v0.1.0 에 관하여

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#12] [service]
v0.1.0 에서 돌아가야하는 기능 얘기해보자

**Comment by iamyanghee** [comment]
@iamyanghee 

### 스포티파이 토큰 관련 기능

- spotify client 정보 입력하면 권한 승인받아 refresh_token 까지 받아오도록 :
  spotify client 정보 입력 받아 → db 에 적재 → 해당 client 정보로 refresh_token 받아와 → db 에 적재

- spotify api 콜하는 곳에서 만료되지않은 access_token 받아와서 쓸 수 있도록 :

**Comment by bksooon** [comment]
@bksooon 

## 메론차트만 띄우기

**Comment by Lucete28** [comment]
@Lucete28 

# 검색시 관련 데이터 적재

main 
이양희 자문위원회장
손병규 파이썬 과외 선생님
한정석 api 통합 위원회
손준성 친한 동생 a.k.a 베프
박기범 님이 주는 정보 엔드 포인트로 넘기기

sub
가사데이터 크롤링 및 적재
정석님 api 통합 
ml을 위한 데이터 준비
    1. 가사 특성 분석
        1-1. 단어 라벨링

**Comment by json9101** [comment]
@json9101 

- 회원가입 완료하면 데이터베이스에 있으면 이미 등록된 email이라고 뜨고 없으면 홈페이지로 넘어가기
- 비로그인 서비스들 완료

**Comment by jungssg** [comment]
- 검색어 입력받으면 spotify search api 로 type = artist, album, track 검색한 결과 return(화면 구성용)
- 검색결과에서 특정 곡, 앨범, 가수 클릭시 상세정보 search 후 DB 적재
- 차트에 포함된 곡 정보(가수, 앨범) DB 적재
